specs/v2-sketch.yaml

```diff
@@ -0,0 +1,375 @@
+          description: An object with workflow information
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Workflow"
+  /workflow/{id}/jobs:
+    get:
+      summary: Return all the known jobs of a given workflow
+      operationId: GET workflow jobs by ID
+      parameters:
+        - name: id
+          in: path
+          description: ID of the workflow
+          required: true
+          schema:
+            type: string
+            format: uuid
+        - name: page_token
+          in: query
+          description: Page token
+          schema:
+            type: string
+        - name: page_size
+          in: query
+          description: Number of jobs to return per page of results, default 50
+          schema:
+            type: number
+            format: integer
+      responses:
+        '200':
+          description: a paginated set of a workflow's jobs as an inverted adjacency list
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  next_page_token:
+                    type: string
+                  jobs:
+                    type: array
+                    items:
+                      $ref: "#/components/schemas/Job"
+components:
+  schemas:
+    Me:
+      # minimise breaking changes from /api/v1.1/me
+      # See https://github.com/circleci/circle/blob/gordon/disable-build-archive/src/circle/model/user.clj#L755-L765
+      type: object
+      properties:
+        login:
+          type: string
+        created_at:
+          type: string
+          format: date-time
+        name:
+          type: string
+        # no idea what `admin` really is
+        #admin:
+        # type: boolean?
+        basic_email_prefs:
+          type: string
+          enum: ["all", "smart", "none"]
+          # This is literally a map of vcs_url to {"on_dashboard": boolean, "emails": "default"}
+        projects:
+          type: object
+          example: {"https://github.com/username/my_project": {"on_dashboard": true, "emails": "smart"}}
+          additionalProperties:
+            type: object
+            properties:
+              on_dashboard:
+                type: boolean
+              emails:
+                type: string
+                enum: ["all", "smart", "none", "default"]
+        organization-prefs:
+          type: object
+          example: {"github": {"username": {"email": "username@example.com"}}}
+          additionalProperties:
+            type: object
+            additionalProperties:
+              type: object
+              properties:
+                email:
+                  type: string
+        trial-end:
+          type: string
+          format: date-time
+        plan:
+          type: object
+          description: I am not certain what this looks like, I'm somewhat afraid it's a Plan document.
+          deprecated: true
```

Comment:
I think we should just remove this. It's always `nil`, at least for all new users.

```
user=> (->> (circle.model.user/all :limit 100000 :sort {:_id -1} :only [:plan]) (map (comp some? :plan)) frequencies clojure.pprint/pprint)
{false 100000}
```

Reply:
Removed